microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -1050,6 +1054,51 @@ If your device is managed by using group policies set by your organization, the
 Text prediction is only available in select languages and regions.
 
 
+<!-- ------------------------------ -->
+#### Windows Ink
+
+Windows Ink allows you to write with a digital pen directly into an input field on Windows touch devices.  This helps you stay in your workflow by reducing the need to switch back and forth between the keyboard and a digital pen.  Supported fields include text boxes, `contenteditable` elements, and the Address bar.  Password fields are not supported.
+
+The following writing tasks are supported:
+
+<!-- todo: 'task' vs. 'action', in context of whitepaper rather than design doc
+The terminology usage could be either:
+* a task is accomplished by an action/gesture
+* an action is accomplished by a gesture
+-->
+
+| Task | Action | Before | After |
+|---|---|---|---|
+| Insert text | Write in or near the text box. | ![Insert text (before)](./index-images/Abydos-add-text-before.png) | ![Insert text (after)](./index-images/Abydos-add-text-after.png) |
+| Delete text | Scribble over the words that you want to delete. | ![Delete text (before)](./index-images/Abydos-delete-text-before.png) | ![Delete text (after)](./index-images/Abydos-delete-text-after.png)|
+| Add space | Draw a vertical line where you want a space to be inserted. | ![Add space (before)](./index-images/Abydos-insert-space-before.png) | ![Add space (after)](./index-images/Abydos-insert-space-after.png) |
+| Remove space | Draw a vertical line on the space that you want to remove. | ![Remove space (before)](./index-images/Abydos-remove-space-before.png) | ![Remove space (after)](./index-images/Abydos-remove-space-after.png) |
+| Select text | Draw a horizontal line across the text to select. | ![Select text (before)](./index-images/Abydos-select-text-before.png) | ![Select text (after)](./index-images/Abydos-select-text-after.png) |
+| Add newline | Draw a "backwards L" at the end of a line of text. | ![Add newline (before)](./index-images/Abydos-add-newline-before.png) | ![Add newline (after)](./index-images/Abydos-add-newline-after.png) |
+<!-- todo: rename pngs lowercase and not Abydos product code name per style guide: https://review.learn.microsoft.com/help/platform/seo-optimize-urls?branch=main#name-your-files -->
```

Comment:
I think this table would be more appropriate for a Windows Ink specific webpage rather than the public whitepaper. I suggest simplifying this to a simple overview of the feature.

```suggestion
Windows Ink supports the following writing tasks:

- Insert text
- Delete text
- Add space
- Remove space
- Select text
- Add newline

```

Reply:
if we elim. table, that gets rid of my terminology question about 'task' vs. 'action' vs. 'gesture' (eg for column 1 & 2 headers)

Reply:
if we elim the pngs from table, obviates renaming the pngs.

Reply:
I'm actually not able to find any public docs with this detail for Windows Ink :( Ok it we keep this table in for now until we find a new home for the info?

Reply:
kept table, used the new lead-in
Windows Ink supports the following writing tasks: